docs/docs/pricing/README.md

```diff
@@ -5,9 +5,9 @@ next: false
 
 # Plans and Pricing
 
-We believe anyone should be able to run simple, low-volume workflows at no cost. We also hope that you share your [sources](/components#sources), [workflows](/workflows), [actions](/components#actions), and other integration components so that other Pipedream users benefit from your work.
+**Pipedream offers a generous free tier**. We believe anyone should be able to run simple, low-volume workflows at no cost. We also hope that you share your [sources](/components#sources), [workflows](/workflows), [actions](/components#actions), and other integration components so that other Pipedream users benefit from your work.
```

Comment:
the 2nd sentence feels a bit out of place here IMO. it doesn't really have to do w/ the free tier, and it's also not clear how to share source/actions/workflows.